longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
+}
+
+variable "redis_type" {
+  default = "hobby-dev"
+}
+
+variable "database_name" {}
+
+variable "database_type" {
+  default = "db.t2.medium"
+}
+
+variable "database_size" {
+  default     = 100
+  description = "The amount of storage to allocate to the database, in GB."
+}
+
+variable "email_name" {}
+variable "email_address" {}
+
+variable "papertrail_destination" {}
+
+data "aws_ssm_parameter" "database_username" {
+  name = "/${var.name}/rds/username"
+}
+
+data "aws_ssm_parameter" "database_password" {
+  name = "/${var.name}/rds/password"
+}
+
+data "aws_ssm_parameter" "mandrill_api_key" {
+  name = "/mandrill/api-key"
+}
+
+resource "heroku_app" "app" {
+  name   = "dosomething-${var.name}"
+  region = "us"
+
+  organization {
+    name = "dosomething"
+  }
+
+  config_vars {
+    # App settings:
+    APP_ENV   = "production"
+    APP_DEBUG = "false"
```

Comment:
debug should be conditional on env, if prod then false if dev then true.

Reply:
We want to enforce this setting for `APP_DEBUG` on all instances that are publicly accessible, since Laravel uses this to determine whether to display full stack traces & environment dump on errors. If we set `APP_DEBUG` to "true", we'd give away secrets!